packages/definitions/dist/fhir/r4/profiles-medplum.json

```diff
@@ -823,6 +823,21 @@
               "min" : 0,
               "max" : "1"
             }
+          },
+          {
+            "id" : "ClientApplication.refreshTokenExpiry",
+            "path" : "ClientApplication.refreshTokenExpiry",
+            "definition" : "Optional configuration to set the refresh token expiry",
+            "min" : 0,
+            "max" : "1",
+            "type" : [{
+              "code" : "date"
```

Comment:
for our use case we'd like to specify a value here which is relative to the current time, not a fixed date parameter.

the expiration value is passed to the [`jose` npm package](https://github.com/panva/jose), which treats `number` and `Date` values as the fixed expiration time in the claim ([reference here](https://github.com/panva/jose/blob/4261556a123ae2dc5c5f238465eff7eb9404b293/src/jwt/produce.ts#L103-L105)), however `string` values are converted to a time span relative to the current time ([reference here](https://github.com/panva/jose/blob/4261556a123ae2dc5c5f238465eff7eb9404b293/src/jwt/produce.ts#L106-L115)) - which matches the existing behaviour with the `2w` default.

---

i think ideally this property would be a `string` which could either support the [full range of jose formats](https://github.com/panva/jose/blob/4261556a123ae2dc5c5f238465eff7eb9404b293/src/jwt/produce.ts#L109-L115), or perhaps a subset of this (to allow for easier validation in the medplum middleware)

Reply:
Yes I think if we can change the name of this field to `refreshTokenLifetime` and make it an expiry duration as a string (eg. '2w', '1h', etc.) that maps to `jose` formats as @josh- pointed out, that would be ideal 👍

Reply:
@thejwuscript  - just wanted to make sure you saw the above comment from @ThatOneBro . This is our main feedback on the PR

Reply:
Yes, thanks very much for the feedback. I'll be making the changes.

Reply:
@thejwuscript do you know when you'll be able to make those changes? let me know if I can help. eager to be able to use this.

Reply:
@josh- I'll try get it done by Monday.